README.txt

```diff
@@ -0,0 +1,25 @@
+# wallpapers
```

Comment:
I think you can drop the .txt file now that we have the .md one (markdown enabled)

Reply:
This `README.txt` file is copied via `make-zip.sh` to `%SystemRoot%\Web\Wallpaper`.

Reply:
Isn't there a command that tells to the zip program to add a file with a different name? If so maybe tell it to add that `README.md` file using the `README.txt` name.

Reply:
No such an option, but we can still use a temporary filename.

Reply:
Done.